docs/tutorial.md

```diff
@@ -49,11 +57,32 @@ get "/" do
 end
 ```
 
-## Run your awesome web project.
+All the code should look like this:
+
+```ruby
+require "./crystal_test/*"
+require "kemal"
+
+module AwesomeWebProject
+  get "/" do
+    "Hello World!"
+  end
+end
+```
+
+## 5. Run your awesome web project.
 
 ```
 crystal build --release src/awesome_web_project.cr
 ./awesome_web_project
 ```
```

Comment:
I think we should also mention magic param `_method` explicitly to make documentation more clearer.